plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/github.ts

```diff
@@ -155,77 +161,106 @@ export function createPublishGithubAction(options: {
         username: owner,
       });
 
-      const repoCreationPromise =
-        user.data.type === 'Organization'
-          ? client.repos.createInOrg({
-              name: repo,
-              org: owner,
-              private: repoVisibility === 'private',
-              visibility: repoVisibility,
-              description: description,
-            })
-          : client.repos.createForAuthenticatedUser({
-              name: repo,
-              private: repoVisibility === 'private',
-              description: description,
-            });
-
-      const { data } = await repoCreationPromise;
-      if (access?.startsWith(`${owner}/`)) {
-        const [, team] = access.split('/');
-        await client.teams.addOrUpdateRepoPermissionsInOrg({
-          org: owner,
-          team_slug: team,
-          owner,
-          repo,
-          permission: 'admin',
-        });
-        // no need to add access if it's the person who own's the personal account
-      } else if (access && access !== owner) {
-        await client.repos.addCollaborator({
-          owner,
-          repo,
-          username: access,
-          permission: 'admin',
-        });
-      }
+      let exists = false;
+      if (ctx.input.skipIfExists) {
```

Comment:
It would be cleaner if the logic inside this if statement would return early instead of setting `exists = true` which is checked later on.

Reply:
I removed the exists variable and return instead.